tests/sdk.test.js

```diff
@@ -0,0 +1,357 @@
+const chai = require('chai');
+chai.use(require('sinon-chai'));
+const { expect } = chai;
+const axios = require('axios');
+const { errors } = require('../lib/utils');
+const rewire = require('rewire');
+const { createProxyServer, createServer } = require('./utilities/mockServer');
+
+const testApiKey =
+  'ev:key:1:3bOqOkKrVFrk2Ps9yM1tHEi90CvZCjsGIihoyZncM9SdLoXQxknPPjwxiMLyDVYyX:cRhR9o:tCZFZV';
+const testAppId = 'app_8022cc5a3073';
+const testKey = 'legacy-key';
+const testEcdhCageKey = 'AjLUS3L3KagQud+/3R1TnGQ2XSF763wFO9cd/6XgaW86';
+
+describe('evervault client', () => {
```

Comment:
This doesn't cover every case in `client.test.js` but is a starting point for future tests.